homeassistant/components/greeneye_monitor/sensor.py

```diff
@@ -315,6 +318,7 @@ class VoltageSensor(GEMSensor[greeneye.monitor.Monitor]):
 
     _attr_icon = VOLTAGE_ICON
```

Comment:
We don't need a custom icon when setting the device class. The frontend already sets a custom icon then.
```suggestion
```